odb_deploy.sh

```diff
@@ -483,15 +478,16 @@ function startServer() {
 	return 1
     fi
 
-    # Start it.
-    ssh "$SSH_CONNECT" "cd $REMOTE_SPDB_PATH && nohup ./$REMOTE_SPDB_CMD > sysout.txt 2>&1"
+    # Start it, but do so in the background. This is because for some odd reason, the ssh command
+    # does not always return here.
+    ssh -f "$SSH_CONNECT" "cd $REMOTE_SPDB_PATH && ./$REMOTE_SPDB_CMD > sysout.txt 2>&1 &"
     if [[ $? -ne 0 ]]; then
-	logError "startServer: could not start server."
-	return 1
+	logWarning "startServer: could not confirm that server started."
     fi
 
-    logInfo "Waiting 20 seconds for $SERVER to complete startup."
-    sleep 20
+    # We need to wait long enough for the database to come up.
+    logInfo "Waiting 60 seconds for $SERVER to complete startup."
+    sleep 60
```

Comment:
I guess we could do better here, eventually.  There's no guarantee that the ODB starts up successfully in 60 seconds.  At the very least you can find the appropriate message in the log.  Better we could provide a servlet or something that returns database availability on demand.